pilot/cmd/pilot-agent/status/ready/probe.go

```diff
@@ -25,10 +25,10 @@ import (
 
 // Probe for readiness.
 type Probe struct {
-	LocalHostAddr       string
-	NodeType            model.NodeType
-	AdminPort           uint16
-	receivedFirstUpdate bool
+	ApplicationPorts []uint16
```

Comment:
Where is `ApplicationPorts` used?

Reply:
It was not added by me. Probably merge issue - I will take a look at the latest and revert if needed. Thanks for the catch.